SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -606,21 +606,30 @@ - (void)handleIncomingEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
         }
     } else if (syncMessage.hasRequest) {
         if (syncMessage.request.type == OWSSignalServiceProtosSyncMessageRequestTypeContacts) {
-            OWSSyncContactsMessage *syncContactsMessage =
```

Comment:
Mostly a whitespace change.

Reply:
Is there a reason we don't want to do the same treatment for group sync requests?

Reply:
`[OWSSyncGroupsMessage buildPlainTextAttachmentDataWithTransaction:]` already takes a transaction and none of its inner workings (e.g. `[OWSGroupsOutputStream writeGroup:]` seem unsafe.